executor/adapter.go

```diff
@@ -86,6 +86,11 @@ var (
 	selectForUpdateRetryDuration        = metrics.PessimisticDMLDurationByAttempt.WithLabelValues("select-for-update", "retry")
 	dmlFirstAttemptDuration             = metrics.PessimisticDMLDurationByAttempt.WithLabelValues("dml", "first-attempt")
 	dmlRetryDuration                    = metrics.PessimisticDMLDurationByAttempt.WithLabelValues("dml", "retry")
+
```

Comment:
We may add some comments to explain about the meaning of `Used` and `Effective`.

Reply:
Also in TxnCtx. These 2 names can be confusing